testing/sblim-wbemcli/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Jirka Dutka <jirka@dutka.net>
+# Maintainer: Jirka Dutka <jirka@dutka.net>
+pkgname=sblim-wbemcli
+pkgver=1.6.3
+pkgrel=0
+pkgdesc="WBEM Command Line Interface for CIMOM access"
+url="https://sourceforge.net/projects/sblim/files/sblim-wbemcli"
+arch="all"
+license="CPL-1.0,EPL-1.0"
```

Comment:
the license is separated using a SPDX identifier.

AND -> When the project is under both licenses
OR -> If the user can pick under which license to use